src/io/powerbi/src/main/scala/PowerBIWriter.scala

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
```

Comment:
OK.. hear me out :-) Would it be possible to implement this just in Python since it's used by your demo notebook?

Reply:
Covered this on the call, will do this once the API dust settles